ports/tracy/portfile.cmake

```
@@ -39,27 +43,22 @@ vcpkg_cmake_install()
 vcpkg_copy_pdbs()
 vcpkg_cmake_config_fixup(PACKAGE_NAME Tracy)
 
-function(tracy_copy_tool tool_name)
-    vcpkg_copy_tools(
-        TOOL_NAMES "tracy-${tool_name}"
-        SEARCH_DIR "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/${tool_name}"
-    )
+function(tracy_copy_tool tool_name tool_dir)
     vcpkg_copy_tools(
-        TOOL_NAMES "tracy-${tool_name}"
-        SEARCH_DIR "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/${tool_name}"
-        DESTINATION ${CURRENT_PACKAGES_DIR}/debug/tools/${PORT}
+        TOOL_NAMES "${tool_name}"
+        SEARCH_DIR "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/${tool_dir}"
     )
 endfunction()
 
 if("cli-tools" IN_LIST FEATURES)
-    tracy_copy_tool(capture)
-    tracy_copy_tool(csvexport)
-    tracy_copy_tool(import-chrome)
-    tracy_copy_tool(import-fuchsia)
-    tracy_copy_tool(update)
+    tracy_copy_tool(tracy-capture capture)
+    tracy_copy_tool(tracy-csvexport csvexport)
+    tracy_copy_tool(tracy-import-chrome import)
+    tracy_copy_tool(tracy-import-fuchsia import)
+    tracy_copy_tool(tracy-update update)
```

Couldn't we just use
```suggestion
    vcpkg_copy_tools(
        TOOL_NAMES tracy-capture tracy-csvexport tracy-import-chrome tracy-import-fuchsia tracy-update)
        SEARCH_DIR "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/${tool_name}"
        DESTINATION ${CURRENT_PACKAGES_DIR}/debug/tools/${PORT}
    )
```

---

As far as I know vcpkg_copy_tools does not search recursively in directories. Every tool is in a different directory (the second option in `tracy_copy_tool`) and therefore a separate tool copy call. (Only the two import tools may could be merged). Please correct me if it is somehow possible to search over multiple directories.